lib/utils/import/index.js

```diff
@@ -0,0 +1,49 @@
+import { isEmpty, get, pick } from 'lodash';
+
+const propertyWhitelist = [
+  'content',
+  'creationDate',
+  'lastModified',
+  'modificationDate',
+  'pinned',
+  'tags',
+  'deleted',
+];
+
+const importNotes = (notes = {}, noteBucket) => {
+  if (isEmpty(notes)) {
+    throw new Error('No notes to import.');
+  }
+
+  if (!get(notes, 'activeNotes') || !get(notes, 'trashedNotes')) {
+    throw new Error('Invalid import format: No active or trashed notes found.');
+  }
+
+  const importNote = note => {
+    const importedNote = pick(note, propertyWhitelist);
+
+    // We don't want to allow these properties to be imported, but they need to be set
+    importedNote.publishURL = '';
+    importedNote.shareURL = '';
+    importedNote.systemTags = [];
+
+    importedNote.deleted = get(importedNote, 'deleted', false);
```

Comment:
The `deleted` prop isn’t set on trashed notes in the export json, so nothing is being set to `true` here. We will have to explicitly set this prop to `true` with each note in trashedNotes.

Reply:
Should we add `deleted` prop to the export?

Reply:
The `deleted` property is required, or else Simperium will reject it.

Reply:
I'm guessing that the `deleted` prop was not in the export because having that and also keeping a trashedNotes array would be redundant. We could flatten the export structure, but that would be disruptive for users who have export files with the old structure.

Explicitly setting back the `deleted` prop in the importer side should be ok, I think.